### DISCUSSION TITLE
[委托]寻找真相！

### DISCUSSION THREAD

**Discussion by hanyueninfeng123** [share121/inter-knot#653] [委托]
![4ca7ca9a0197607a](https://github.com/user-attachments/assets/1b2f3996-da25-41da-b8a1-d32c376fa950)

没想到所谓的「实习生配置事故」，揭下面具依然是残酷的商业竞争…去空洞里绑架邦布商人实在是非常恶劣啊！
难怪那位实习生…哦，现在已经是正式员工了！难怪那位商会正式员工，提到竞争对手总是如临大敌。
可恶可恶可恶！有人记得我之前发帖吐槽的，商会里的好几个邦布前辈们因为实习生把萝卜地图配错，纷纷走失在空洞里的事情吗？
那个实习生配置错误后，还没等公司追责就直接跑路了！
啊啊啊啊啊啊好气！因为他和我是一起入职的…我还请他吃过「煮釜」火锅啊！真有人能坏到吃了别人请的火锅还甩锅跑路吗？？
虽然商会的大家说没有严重损失的话，即使追责也很难得到赔偿…但我自己还是气不过！就干脆找人查了一下他…·
结果！结果！那个「实习生」！好像是个收钱办事的盗洞客啊啊啊啊！求一个大佬帮我去他们的窝点走一趟！帮我把他抓回来！我要问问怎么回事，就当是为了麻辣牛肉和什锦丸子呜呜呜呜呜呜··

**Comment by YLMikodo** [comment]
这个委托我接了！

**Reply by hanyueninfeng123** [reply]
谢谢大佬

**Comment by Hongxuli2006** [comment]
我也想接

**Comment by Xmug** [comment]
什么？有吃的？！

**Comment by 2132925099** [comment]
番茄火锅才是王中王！

**Comment by hanyueninfeng123** [comment]


**Reply by aiTunes-dev** [reply]
是滴

**Reply by Mu-7593** [reply]
昨天就有很多人用动图发帖了，只是很多都沉了现在首页看不到

**Comment by Mu-7593** [comment]
有点意思，坐等后续

**Comment by Demise-deity** [comment]
什么吃的，好吃吗，我也要吃

**Comment by GanlinSama** [comment]
蹲一手后续，这事还挺抽象的

**Comment by Pai-pi** [comment]
经验+3

**Comment by yongko2006** [comment]
饿了

**Comment by KSN-tt41** [comment]
蛮有意思的，有后续踢我

**Comment by QingMingqwq** [comment]
蹲个后续

**Comment by aaa00073** [comment]
吃瓜群众，这事挺有意思，蹲一手看有没有后续事件，期待

**Comment by ChevalierDeSangreal** [comment]
插眼蹲一个后续

[a comment by an automated account is not shown]

**Comment by Y-0930** [comment]
会不会是对手公司干的
![v2-2155390f2c03e19505811175e4320e16_r](https://github.com/user-attachments/assets/23baf83b-6aa8-4f8b-9772-e692cfd0f67f)

**Comment by Amzen10086** [comment]
这个委托我接了，我可是好评率99%的金牌绳匠

**Comment by kittyll1** [comment]
蹲一手后续
